doc/amazon_sagemaker_processing.rst

```diff
@@ -27,37 +27,38 @@ You can run notebooks on Amazon SageMaker that demonstrate end-to-end examples o
 Data Pre-Processing and Model Evaluation with Scikit-Learn
 ==================================================================
 
-You can run a Scikit-Learn script to do data processing on SageMaker using the `SKLearnProcessor`_ class.
-
-.. _SKLearnProcessor: https://sagemaker.readthedocs.io/en/stable/sagemaker.sklearn.html#sagemaker.sklearn.processing.SKLearnProcessor
+You can run a Scikit-Learn script to do data processing on SageMaker using the :class:`sagemaker.sklearn.processing.SKLearnProcessor` class.
 
 You first create a ``SKLearnProcessor``
 
 .. code:: python
 
     from sagemaker.sklearn.processing import SKLearnProcessor
 
-    sklearn_processor = SKLearnProcessor(framework_version='0.20.0',
-                                     role='[Your SageMaker-compatible IAM role]',
-                                     instance_type='ml.m5.xlarge',
-                                     instance_count=1)
+    sklearn_processor = SKLearnProcessor(
+        framework_version='0.20.0',
+        role='[Your SageMaker-compatible IAM role]',
+        instance_type='ml.m5.xlarge',
+        instance_count=1,
+    )
 
 Then you can run a Scikit-Learn script ``preprocessing.py`` in a processing job. In this example, our script takes one input from S3 and one command-line argument, processes the data, then splits the data into two datasets for output. When the job is finished, we can retrive the output from S3.
 
 .. code:: python
 
     from sagemaker.processing import ProcessingInput, ProcessingOutput
 
-    sklearn_processor.run(code='preprocessing.py',
-                      inputs=[ProcessingInput(
-                        source='s3://your-bucket/path/to/your/data,
-                        destination='/opt/ml/processing/input')],
-                      outputs=[ProcessingOutput(output_name='train_data',
-                                                source='/opt/ml/processing/train'),
-                               ProcessingOutput(output_name='test_data',
-                                                source='/opt/ml/processing/test')],
-                      arguments=['--train-test-split-ratio', '0.2']
-                     )
+    sklearn_processor.run(
+        code='preprocessing.py',
+        inputs=[
+            ProcessingInput(source='s3://your-bucket/path/to/your/data, destination='/opt/ml/processing/input'),
```

Comment:
`@ignorable` nit-doc: should probably match our preferred formatter `black` with double-quotes, etc.